rdr_service/alembic/versions/bd7e69e0e71c_profile_update_archive_table.py

```diff
@@ -0,0 +1,65 @@
+"""profile_update archive table
+
+Revision ID: bd7e69e0e71c
+Revises: b7971e81863c
+Create Date: 2022-11-30 14:35:09.964277
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import rdr_service.model.utils
+
+
+from rdr_service.participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from rdr_service.participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from rdr_service.participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from rdr_service.participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from rdr_service.participant_enums import MetricSetType, MetricsKey, GenderIdentity
+from rdr_service.model.base import add_table_history_table, drop_table_history_table
+from rdr_service.model.code import CodeType
+from rdr_service.model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = 'bd7e69e0e71c'
+down_revision = 'b7971e81863c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('profile_update',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
```

Comment:
Perhaps on a follow up PR, we can add a `modified` column as well?  There are reasons that having `id, created, modified `as standard columns is beneficial for the RDR-to-PDR pipeline, because any tables that have those three columns (especially modified) can be handled by a generic process that looks for recently modified records.